js/js-unit-testing/js-unit-testing_people/index.test.js

```diff
@@ -48,22 +48,76 @@ test("getFirstNames() returns an array with the first names", () => {
   expect(result).toEqual(["Jane", "John", "Max", "Erika", "Luke"]);
 });
 
-test('getFullNames() returns an array with full names as strings, e.g. "Jane Doe"', () => {});
+test('getFullNames() returns an array with full names as strings, e.g. "Jane Doe"', () => {
+  const result = getFullNames(people);
+  expect(result).toEqual([
+    "Jane Doe",
+    "John Doe",
+    "Max Mustermann",
+    "Erika Musterfrau",
+    "Luke Skywalker",
+  ]);
+});
 
-test('getNameAndAge() returns an array of formatted strings like this: "Skywalker (23)"', () => {});
+test('getNameAndAge() returns an array of formatted strings like this: "Skywalker (23)"', () => {
+  const result = getNameAndAge(people);
+  expect(result).toEqual([
+    "Doe (21)",
+    "Doe (18)",
+    "Mustermann (32)",
+    "Musterfrau (38)",
+    "Skywalker (23)",
+  ]);
+});
 
-test("getPeopleByAge() returns an array of people of a certain age", () => {});
+test("getPeopleByAge() returns an array of people of a certain age", () => {
+  const result = getPeopleByAge(people, 23);
+  expect(result).toEqual([
+    "Max Mustermann",
+    "Erika Musterfrau",
+    "Luke Skywalker",
+  ]);
+});
 
-test('getPeopleNamesOlderThan() returns an array of people full names older than a certain age, e.g. "Luke Skywalker"', () => {});
+test('getPeopleNamesOlderThan() returns an array of people full names older than a certain age, e.g. "Luke Skywalker"', () => {
+  const result = getPeopleNamesOlderThan(people, 23);
+  expect(result).toEqual(["Max Mustermann", "Erika Musterfrau"]);
+});
 
-test("getPeopleByLastName() returns an array of people of a certain lastName", () => {});
+test("getPeopleByLastName() returns an array of people of a certain lastName", () => {
+  const result = getPeopleByLastName(people, "Mustermann");
+  expect(result).toEqual(["Mustermann"]);
+});
 
 // Hint: use the array method find() for the implementation
-test("findPersonById() returns the full object of a person", () => {});
+test("findPersonById() returns the full object of a person", () => {
+  const result = findPersonById(people, 5);
+  expect(result).toEqual({
+    id: 5,
```

Comment:
another option could be "expect(result).toEqual(people[4])" (instead of copying the whole object, but for readability maybe this is actually much nicer how you have done it!)